objectRecognitionSource/README.md

```diff
@@ -39,6 +39,8 @@ Additionally, an example VANTIQ project named *objRecExample.zip* can be found i
     *   [YOLO Processor](#yoloNet) -- An implementation of the [You Only Look Once](https://pjreddie.com/darknet/yolo/)
         (YOLO) object detection software using Java Tensorflow.
     * [YOLO Models Available](#modelFiles)
+    * [No Processor](#noNet) -- An implementation of the interface used only to **save** images. This implementation does 
+    **no** image processing.
```

Comment:
maybe "... use to just **save** images."

(my concerns is that "used only to save images" makes it sound like this is used to save images when saving them.  Just trying to reduce confusion.